### DISCUSSION TITLE
駭客對俄國網站發出爆量請求(request)

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS3073-C/Python-Community-GS3073-C#297] [Q&A] [answered]
之前有看到一則新聞，內容是說有資訊工程師在每天上班前，都會對俄羅斯網站發出超級多(爆量)的請求，讓此俄羅斯網站無法運作。這件事聽起來蠻簡單的，是否就是老師上課時說的「注入攻擊(SQL injection)」呢？注入攻擊會很難做到嗎?反過來講，「注入攻擊」會很難阻擋嗎?因為人們不是很久以前就知道有注入攻擊，難道網站沒有任何防禦這些攻擊的方式嗎?

**Comment by fatsoap** [comment, marked as answer]
發出超級多(爆量)的請求 -> 這應該是 DDOS 吧

**Reply by enoch-lightning** [reply]
我查到下面這個
[什麼是 DDoS 攻擊？](https://aws.amazon.com/tw/shield/ddos-attack-protection/)
看起來是蠻難的東西。
